### ISSUE TITLE
Question: inputmask-date cannot remove placeholder

### ISSUE THREAD

**Issue by niels-numbers** [RobinHerbots/Inputmask#1801]
I want to restrict a field to date format (dd/mm/yyyy).  This is my code:

```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title></title>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    <link href="css/inputmask.css" rel="stylesheet"/>
  <script src="dist/jquery.inputmask.bundle.js" ></script>
  </head>
  <body>

      <input type="text" class="form-control inputConstrains" name='bezahltAm' data-inputmask="'alias': 'dd/mm/yyyy'" data-mask>

    <script>
      $("[data-mask]").inputmask();
    </script>

  </body>
</html>
```
This is how it looks:

![image](https://user-images.githubusercontent.com/1765602/35336926-240affb8-011a-11e8-8188-73d57e94dbac.png)

However,   I cannot change/remove the placeholder. What am I missing?

**Comment by RobinHerbots** [comment]
@iwasherefirst2 , 

Can you explain a bit more the problem.  You don't want to see the placeholder?  Or you can't type anything?

**Comment by niels-numbers** [comment]
Thank you for your quick answer RobinHerbots.

The problem is that I cannot type anything. If I hit backspace then nothing happens.
![image](https://user-images.githubusercontent.com/1765602/35379692-2f6e5c9a-01b7-11e8-92f3-d4fdf69cc66e.png)

If I select anything and try to replace it by hitting a key then nothing happens. If I move the courser to the input field I just get
![image](https://user-images.githubusercontent.com/1765602/35379732-56a36efe-01b7-11e8-9ce9-16452d44317c.png)
and its unchangeable. By the way, it does not look like a placeholder to me, because whenever I move the courser outside the  input field, its empty.

I tried it on Ubuntu 16.04 with Firefox and Chrome.

**Comment by RobinHerbots** [comment]
@iwasherefirst2 , 

Have you tried the datetime on the demo page?

**Comment by niels-numbers** [comment]
@RobinHerbots yes that works perfectly:

![image](https://user-images.githubusercontent.com/1765602/35385023-5f8e6ce8-01c7-11e8-8b31-85abc3153a32.png)

**Comment by niels-numbers** [comment]
So all I did is I downloaded the current master branch as zip, extracted it and created a text.html with the above content. Did I miss any step? 

I also checked that the files are actually loaded:

![image](https://user-images.githubusercontent.com/1765602/35385248-f78b1406-01c7-11e8-8797-ab41d776b206.png)

![image](https://user-images.githubusercontent.com/1765602/35385297-250d0ab0-01c8-11e8-880d-14c8c748225f.png)

**Comment by RobinHerbots** [comment]
@iwasherefirst2 , 

Hmm that should work, ... can you reproduce this in a jsfiddle?

**Comment by niels-numbers** [comment]
I did a jsfiddle and it worked fine.

I realized that my code works if I include the inputmask.bundle.js by cdn : 

`<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.inputmask/3.3.4/jquery.inputmask.bundle.min.js"   ></script>`

But it does not work if I include the bundle.js from the current git repository:
`<script src="dist/jquery.inputmask.bundle.js" ></script>`

Is maybe something wrong in the current git version?

**Comment by RobinHerbots** [comment]
@iwasherefirst2 , 

Can you try with https://rawgit.com/RobinHerbots/Inputmask/4.x/dist/jquery.inputmask.bundle.js

**Comment by iwasherefirst2** [comment]
I tried it, its not working with the  https://rawgit.com/RobinHerbots/Inputmask/4.x/dist/jquery.inputmask.bundle.js, but it does work with https://cdnjs.cloudflare.com/ajax/libs/jquery.inputmask/3.3.4/jquery.inputmask.bundle.min.js

**Comment by niels-numbers** [comment]
Here is a jsFiddle with the problem: https://jsfiddle.net/u0r9o45q/

**Comment by RobinHerbots** [comment]
@iwasherefirst2 , 

Of course, ... have a look at https://jsfiddle.net/u0r9o45q/2/

In the new 4.x branch I removed the dd/mm/yyyy alias.  It is replaced by a datetime alias where you can specify the date format.  I need to update the readme for datetimes in the 4.x branch

What you saw was a static mask with the dd/mm/yyyy and so not editable.

**Comment by niels-numbers** [comment]
Alright, thank you for letting me know :+1: !

**Comment by Daughty** [comment]
> @RobinHerbots ,
> 
> Of course, ... have a look at https://jsfiddle.net/u0r9o45q/2/
> 
> In the new 4.x branch I removed the dd/mm/yyyy alias. It is replaced by a datetime alias where you can specify the date format. I need to update the readme for datetimes in the 4.x branch
> 
> What you saw was a static mask with the dd/mm/yyyy and so not editable.

----> Is this version of change is available for bower components as well?
